src/Illuminate/Foundation/Http/Middleware/VerifyPostSize.php

```diff
@@ -0,0 +1,53 @@
+<?php namespace Illuminate\Foundation\Http\Middleware;
+
+use Closure;
+use Illuminate\Contracts\Routing\Middleware;
+use Illuminate\Http\Exception\PostTooLargeException;
```

Comment:
Or alternatively move this to `illuminate/http` so that it can be used with lumen framework.